docs/introduction.mdx

```diff
@@ -26,6 +26,8 @@ Pycord is an advanced and complex Python library. To take advantage of the full
 
 This list is not exhaustive, but it should give you a good idea of what you need to know to get started.
 
+When you see ellipsis (that's the three dots!) in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!
```

Comment:
```suggestion
When you see ellipsis – that's the three dots! – in this guide, it is a placeholder for code that you need to fill in. Sometimes, it may be an entire command's code, while other times it could be something simple like a Guild ID. Make sure to replace them before running your bot!
```